geant4.spec

```diff
@@ -49,7 +49,6 @@ cmake ../%{n}.%{realversion} \
   -DGEANT4_BUILD_TLS_MODEL:STRING="global-dynamic" \
   -DGEANT4_ENABLE_TESTING=OFF \
   -DGEANT4_BUILD_VERBOSE_CODE=OFF \
-  -DGEANT4_BUILD_BUILTIN_BACKTRACE=OFF \
```

Comment:
@iarspider , if I understad the @civanch comment https://github.com/cms-sw/cmsdist/issues/7734 , then we only need to remove the changes of vecgeom.spec. So I think this change we still need ... right @civanch ? I might be wrong, so please wait for @civanch response